dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EntryResource.java

```diff
@@ -133,6 +137,19 @@ public Entry addAliases(@ApiParam(hidden = true) @Parameter(hidden = true, name
         return this.toolDAO.findCollectionsByEntryId(entry.getId());
     }
 
+    // TODO: Write tests for this endpoint.
+    @GET
+    @Path("/{entryId}/verifiedPlatforms")
+    @UnitOfWork
+    @Operation(operationId = "getVerifiedPlatforms", description = "Get the verified platforms for each version", security = @SecurityRequirement(name = OPENAPI_JWT_SECURITY_DEFINITION_NAME))
+    public List<VersionVerifiedPlatform> getVerifiedPlatforms(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth Optional<User> user,
+            @Parameter(name = "entryId", description = "id of the entry", required = true, in = ParameterIn.PATH) @PathParam("entryId") long entryId) {
```

Comment:
entryId should be a Long.

1. At a quick glance, it seems to be a Long in the other resource file I looked it.
2. There used to be a thing in older JDKs, that may not apply any more, but I still do it, where you shouldn't let Java do autoboxing (automatically convert primitive to object and vice-versa), and should instead do it explicitly -- your long gets "autoboxed" into a Long on the next line. So just make it a Long to start with.